### DISCUSSION TITLE
CIPP authorization for a GDAP only customer

### DISCUSSION THREAD

**Discussion by BDOMichaelKirstNeshva** [KelvinTegelaar/CIPP#1398] [Q&A] [answered]
Hi, we have now some new customers with GDAP only access.
These customers are listed inside the partnercenter.
But how can i manage these inside CIPP?
with these customers are no "reseller agreements", only "Consulting" Managed Services. So DAP on-boarding isn't a solution.

Thanks
Michael

**Comment by KelvinTegelaar** [comment, marked as answer]
Microsoft says that the Contract portion of DAP/GDAP will always be mandatory, anything else is not supported. Thus you must create a relationship without Delegated Access Permissions.

**Reply by BDOMichaelKirstNeshva** [reply]
Hi Kelvin,
thanks for your response.

But I wrote, that we have active GDAP relationships.
We are working in some functions as member of "AdminAgents" or customized Groups with delegated permissions.
Every **GDAP only customer** is listed inside https://partner.microsoft.com/en-us/dashboard/commerce2/granularadminaccess/list
but **not in DAP related l**ist  https://partner.microsoft.com/en-us/dashboard/commerce2/customers/list

These GDAP related Tenants are not listed in the "Tenant List" of CIPP
These GDAP customers don't have any DAP relationship, so I can't migrate them. (And the GDAP Migration Tool is out of scope in this discussion)

So, the question is: How can I manage **GDAP only customers** with CIPP?
Is one of the options, to add the App Client ID of the CIPP App inside the group which has GDAP rights at the customer tenant site? (like AdminAgents)
Thanks, Michael
